mllib/src/test/scala/org/apache/spark/ml/regression/GeneralizedLinearRegressionSuite.scala

```diff
@@ -743,6 +743,54 @@ class GeneralizedLinearRegressionSuite
     }
   }
 
+  test("generalized linear regression: intercept only") {
+    /*
+      R code:
+      y <- c(17, 19, 23, 29)
+      w <- c(1, 2, 3, 4)
+      model1 <- glm(y ~ 1, family = poisson)
+      model2 <- glm(y ~ 1, family = poisson, weights = w)
+      as.vector(c(coef(model1), coef(model2)))
+      [1] 3.091042 3.178054
+     */
+
+    val dataset = Seq(
+      Instance(17.0, 1.0, Vectors.zeros(0)),
+      Instance(19.0, 2.0, Vectors.zeros(0)),
+      Instance(23.0, 3.0, Vectors.zeros(0)),
+      Instance(29.0, 4.0, Vectors.zeros(0))
+    ).toDF()
+
+    val expected = Seq(3.091, 3.178)
+
+    import GeneralizedLinearRegression._
```

Comment:
it seems weird to me that several tests do this, import GeneralizedLinearRegression._, and we don't just have one at the top, but it looks like it is unrelated to your changes.